webpack/assets/javascripts/react_app/routes/Settings/SettingsPage/components/TabPaneContent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Table } from '../../../../components/common/table';
+import createSettingsTableSchema from './SettingsTableSchema';
+
+import TestEmail from './TestEmail';
+
+const TabPaneContent = ({ category, settings, onEditClick }) => (
+  <React.Fragment>
+    {category === 'Setting::Email' ? <TestEmail /> : null}
```

Comment:
Do we have any strategy (could be follow-up) to enable plugins adjust their setting pages?
```suggestion
    <Slot [some category-based-identifier of slot>]>
```

Reply:
Not yet, I haven't found a plugin that would need it.